libs/go/zmscli/cli.go

```diff
@@ -46,6 +49,11 @@ type Zms struct {
 	AddSelf          bool
 }
 
+type SuccessMessage struct {
```

Comment:
Struct for success message. For now I'm planning to leave the original text in the "Message" field as it might be in use for users.
Example success Json:
` $ target/darwin/zms-cli -o json  -d home.test add-member admin user.example
{
    "Status": 200,
    "Message": "[Added to admin: user.example]"
}`

Example Yaml:
`$ target/darwin/zms-cli -o yaml -d home.test add-member admin user.example
status: 200
message: '[Added to admin: user.example]'`

Reply:
looks good!